src/Guild.js

```diff
@@ -25,6 +25,29 @@ class Guild {
     autoBind(this);
   }
 
+  getRoleNameByID (value) {
+    return Object.keys(this.roles).find(key => this.roles[key] === value);
+  }
+
+  createRole (id, name) {
```

Comment:
Style: I'd prefer `addRole` as you are adding a Role to the `roles` object, not creating one.

Reply:
Fixed